.gitignore

```diff
@@ -36,4 +36,5 @@ yarn-error.log*
 next-env.d.ts
 
 .idea
-*storybook.log
\ No newline at end of file
+*storybook.log
+pnpm-lock.yaml
```

Comment:
**thought (non-blocking)**: we should probably rework the gitignore to cover different package managers as we don't prescribe a specific one. Happy for your input.

Reply:
Not sure what the best practices are in this case. Are there any?

Reply:
probably not ;) something like https://toptotal.com/developers/gitignore/api/nextjs seems to suggest including npm, yarn, pnpm debug logs but not package lock files.

Reply:
Alright ill make sure to exclude it in the future